example/Cargo.toml

```diff
@@ -9,7 +9,7 @@ bytes = "0.4"
 env_logger = "0.5.4"
 futures = "0.1"
 multiaddr = "0.2"
-multiplex = { path = "../multiplex-rs" }
+libp2p-multiplex = { path = "../multiplex" }
```

Comment:
actually, rename to libp2p-mplex. See the new spec https://github.com/libp2p/mplex